### PR TITLE
Backport of  PR1409

### PR DESCRIPTION
**Pull request by leemyongpakvn**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.x
| Description?  | Wrap bash command in bash block to keep consistency throughout the page and to avoid long command overflow in mobile view.
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
Missing space.

File: basics/installation/localhost.md [hunk 1]
```diff
@@ -28,13 +28,21 @@ Here is a selection of free AMP installers:
 
 To install LAMP on your computer follow these steps (tested on Debian Buster).
 - Update your system
-  - `apt update`
+```bash
+apt update
+```
```
Comment (ghost) [inline basics/installation/localhost.md]: Missing space

Comment (leemyongpakvn) [reply basics/installation/localhost.md]: bash block has left padding for inside command so I don't think we need space there. MySQL commands in bash block a few lines below use the same format. Here is the original PR I made for 8.x docs https://github.com/leemyongpakvn/docs/blob/8.x/basics/installation/localhost.md

Comment (ghost) [reply basics/installation/localhost.md]: There are spaces in the [PR for 8.0](https://github.com/PrestaShop/docs/pull/1409/files#diff-d9a6ca3d6766249de84eeae59807d0a72e6c224e2fdd8f2f45f781b10e174ecdR31-R33)

When i merge your PR i have this 

Screenshot (87)

**Review by ghost — APPROVED** [review]

File: basics/installation/localhost.md (hunk 1 above)
Comment (leemyongpakvn) [reply basics/installation/localhost.md]: I see the missing space clearly now. Thank you.

**Review by NeOMakinG — APPROVED** [review]

**Comment by NeOMakinG** [comment]
Thanks @leemyongpakvn 🐐

**Pull request merged**